src/Shouldly.Tests/ShouldBe/EnumerableType/EnumerableOfStringIsInsensitiveScenario.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.ShouldBe.EnumerableType
+{
+    public class EnumerableOfStringIsInsensitiveScenario : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldPass()
+        {
+            new[] { "foo" }.ShouldBe(new[] { "FOo" }, Case.Insensitive);
+        }
+
+        protected override void ShouldThrowAWobbly()
+        {
+            new[] { "foo" }.ShouldBe(new[] { "different" }, Case.Insensitive, () => "Some additional context");
+        }
+
+        protected override string ChuckedAWobblyErrorMessage
+        {
+            get
+            {
+                return "new[] { \"foo\" } should be [\"different\"] but was [\"foo\"] difference [*\"foo\"*]" +
```

Comment:
I think the error should show if it is a case sensitive or insensitive comparison.